DataConnectors/Zimperium MTD Alerts.json

```diff
@@ -0,0 +1,90 @@
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "ZimperiumThreatLog_CL",
+            "baseQuery": "ZimperiumThreatLog_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All threats with threat vector equal to Device",
+            "query": "ZimperiumThreatLog_CL \n | where threat_vector_s  == \"Device\" \n | limit 100"
+        },
+        {
+            "description" : "All threats for devices running iOS",
+            "query": "ZimperiumThreatLog_CL \n | where device_os_s == \"ios\" \n | order by event_timestamp_s  desc nulls last"
+        },
+        {
+            "description" : "View latest mitigations",
+            "query": "ZimperiumMitigationLog_CL \n | order by event_timestamp_s  desc nulls last"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "ZimperiumThreatLog_CL",
+            "lastDataReceivedQuery": "ZimperiumThreatLog_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        },
+        {
+            "name": "ZimperiumMitigationLog_CL",
+            "lastDataReceivedQuery": "ZimperiumMitigationLog_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "ZimperiumThreatLog_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            }
+        ],
+        "customs":""
+    },
+    "instructionSteps": [
+        {
+            "title": "Configure and connect Zimperium MTD",
+            "description": "1. In zConsole, click **Manage** on the navigation bar.\n2. Click the **Integrations** tab.\n3. Click the **Threat Reporting** button and then the **Add Integrations** button.\n4. Create the Integration:\n  - From the available integrations, select Microsoft Azure Sentinel.\n  - Enter your workspace id and primary key from the fields below, click **Next**.\n  - Fill in a name for your Azure Sentinel integration.\n  - Select a Filter Level for the threat data you wish to push to Azure Sentinel.\n  - Click **Finish**",

```

Comment:
Please add a link to the Zimperium documentation on zConsole specifics.

Reply:
Added link to Connector file.